presto-spark-base/src/main/java/com/facebook/presto/spark/execution/property/NativeExecutionNodeConfig.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spark.execution.property;
+
+import com.facebook.airlift.configuration.Config;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+public class NativeExecutionNodeConfig
+{
+    private static final String NODE_ENVIRONMENT = "node.environment";
```

Comment:
same, let's add comment to indicate the place in C++ where these configs are used